translations/bg.po

```diff
@@ -0,0 +1,332 @@
+msgstr "Увеличи"
+
+msgid "Zoom Reset"
+msgstr "Рестартирай мащаба"
+
+msgid "Zoom Out"
+msgstr "Намали"
+
+msgid "Settings"
+msgstr "Настройки"
+
+msgid "Visuals"
+msgstr "Графики"
+
+msgid "Show Grid"
+msgstr "Покажи решетката"
+
+msgid "Show Handles"
+msgstr "Покажи дръжките"
+
+msgid "Rasterized SVG"
+msgstr "Растеризирай SVG"
+
+msgid "Enable Snapping"
+msgstr "Включи захващането"
+
+msgid "Snap size"
+msgstr "Размер на захващането"
+
+msgid "Copy All Text"
+msgstr "Копирай всичкия текст"
+
+msgid "Import SVG"
+msgstr "Импорт на SVG"
+
+msgid "Export"
+msgstr "Експортирай"
+
+msgid "Couple/Decouple"
+msgstr "Свържи/Отвържи"
+
+msgid "Duplicate"
+msgstr "Дублирай"
+
+msgid "Move Up"
+msgstr "Премести нагоре"
+
+msgid "Move Down"
+msgstr "Премести надолу"
+
+msgid "Delete"
+msgstr "Премахни"
+
+msgid "Insert Before"
+msgstr "Вмъкни отзад"
+
+msgid "Insert After"
+msgstr "Вмъкни отпред"
+
+msgid "Convert To"
+msgstr "Превърни в"
+
+msgid "Doesn’t describe a SVG."
+msgstr "Текстът не описва SVG."
+
+msgid "Improper nesting."
+msgstr "Несъвместими тагове."
+
+msgid "Delete: Deletes the selected tags"
+msgstr "Delete: Изтрива избраните тагове"
+
+msgid "Ctrl+Down: Moves the selected tags down"
+msgstr "Ctrl+Down: Премества избраните тагове надолу"
+
+msgid "Ctrl+Up: Moves the selected tags up"
+msgstr "Ctrl+Up: Премества избраните тагове нагоре"
+
+msgid "Ctrl+D: Duplicates the selected tags"
+msgstr "Ctrl+D: Дублирай избраните тагове"
+
+msgid "Unknown Tag"
+msgstr "Непознат таг"
+
+msgid "Unknown Attribute"
+msgstr "Непознат атрибут"
+
+msgid "Import"
+msgstr "Импортирай"
+
+msgid "Cancel"
+msgstr "Отказ"
+
+msgid "Import Problems"
+msgstr "Проблеми в импортирането"
+
+msgid "GodSVG doesn’t recognize this attribute"
+msgstr "GodSVG не разпознава този атрибут"
+
+msgid "Format"
+msgstr "Формат"
+
+msgid "Scale"
+msgstr "Мащаб"
+
+msgid "Size"
+msgstr "Размер"
+
+msgid "Final size"
+msgstr "Краен размер"
+
+msgid "Export Configuration"
+msgstr "Конфигурация на експорта"
+
+msgid "Add new tag"
+msgstr "Добави нов таг"
+
+msgid "License"
+msgstr "Лиценз"
+
+msgid "Third-party licenses"
+msgstr "Лицензи от трети партии"
+
+msgid "Authors"
+msgstr "Автори"
+
+msgid "Shortcuts"
+msgstr "Бързи клавиши"
+
+msgid "Palettes"
+msgstr "Палети"
+
+msgid "Color Picker"
+msgstr "Цветно колело"
+
+msgid "Add color"
+msgstr "Добави цвят"
+
+msgid "Edit palette name"
+msgstr "Промени името на палетата"
+
+msgid "Unnamed"
+msgstr "Неименуван"
+
+msgid "Edit color name"
+msgstr "Промени името на цвета"
+
+msgid "Delete color"
+msgstr "Изтрий цвета"
+
+msgid "Enable the color"
+msgstr "Включи цвета"
+
+msgid "Disable the color"
+msgstr "Изключи цвета"
+
+msgid "Godot third-party components"
+msgstr "Компоненти в Godot от трети лица"
+
+msgid "Undo"
+msgstr "Върни назад"
+
+msgid "Redo"
+msgstr "Върни напред"
+
+msgid "Copy"
+msgstr "Копирай"
+
+msgid "Paste"
+msgstr "Постави"
+
+msgid "Cut"
+msgstr "Изрежи"
+
+msgid "Alert!"
+msgstr "Предупреждение!"
+
+msgid ""
```

Comment:
Like this, is it normal?

Reply:
Believe it or not, but yes, tho, it's probably better to write it as:
```gettext
msgid "The file couldn't be opened.\n"
"Try checking the file path, ensure that the file is not deleted, or choose a "
"different file."
msgstr "Файлът не може да бъде отворен.\n"
"Опитайте да проверите пътя към файла, уверете се, че файла не е изтрит, или "
"изберете различен файл."
```
So it won't look that confusing

Reply:
It was formatted like that after i opened it in Poedit